Makefile

```diff
@@ -3,7 +3,7 @@ VERSION := $(shell git describe | cut -c2-)
 
 # We cannot use the official tinygo container image until
 # this issue is closed: https://github.com/tinygo-org/tinygo/issues/3501
-CONTAINER_IMAGE = ghcr.io/kubewarden/tinygo/tinygo-dev:0.28.1-multi3_fix
+CONTAINER_IMAGE = ghcr.io/kubewarden/tinygo/tinygo-dev:0.31.1-multi3_fix
```

Comment:
Another thing, I've noticed that something change in the Tinygo side. Because to test the issue I've retriggered a [job](https://github.com/kubewarden/container-resources-policy/actions/runs/7956502381/job/22390731593) that ran successfully before in the main branch and now it's failing.